packages/quantic/force-app/main/default/lwc/quanticTooltip/quanticTooltip.css

```diff
@@ -1,24 +1,57 @@
+:host {
+  --x-translation: var(--adapted-x-translation, 0px);
+  --max-width: var(--adapted-max-width, 20rem);
+}
+
 .tooltip__content {
   left: 50%;
-  transform: translate(-50%, -100%);
+  transition: transform 0.05s linear 0.05s, opacity 0.1s linear 0s, visibility 0.1s linear 0s;
+  z-index: 2;
+}
+
+.tooltip__content--positioned-above {
+  transform: translate(calc(-50% + var(--x-translation)), -100%);
   top: -0.7rem;
 }
 
+.tooltip__content--positioned-below {
+  transform: translate(calc(-50% + var(--x-translation)), 0.7rem);
+}
+
 .tooltip__content--visible {
   visibility: visible;
   opacity: 1;
 }
 
-.tooltip__content--light {
+.tooltip__content--light,
+.slds-popover_tooltip {
   width: max-content;
-  max-width: var(--slds-c-tooltip-sizing-max-width, var(--sds-c-tooltip-sizing-max-width, var(--lwc-sizeMedium, 20rem)));
 }
 
-.slds-nubbin_bottom-left:before,
-.slds-nubbin_bottom-left:after {
+.tooltip__arrow {
   left: 50%;
+  transform: translate(-50%, -100%);
+  transition: opacity 0.08s linear 0s, visibility 0.06s linear 0s;
+  z-index: 2;
 }
 
-.slds-popover_tooltip {
-  width: max-content;
+.tooltip__arrow--dark {
+  background-color: var(--slds-c-tooltip-color-background, rgb(3, 45, 96));
+}
+
+.tooltip__arrow--light {
+  background-color: var(--lwc-colorBackgroundAlt, rgb(255, 255, 255));
+}
+
+.tooltip__arrow--positioned-above {
+  top: -0.7rem;
+}
+
+.tooltip__arrow--positioned-below {
+  transform: translate(-50%, 0.7rem);
```

Comment:
Why is the arrow below translated -50% in X?

Reply:
so it stays in the middle of the tooltip

Reply:
just setting bottom: 0.7rem; doesnt keep it centered like when positioned above?

Reply:
but why isn't it also applied to`tooltip__arrow--positioned-below` that translate -50% in X?

Reply:
css value adjusted.